src/cargo/ops/cargo_compile.rs

```diff
@@ -48,7 +48,7 @@ pub struct CompileOptions<'a> {
     /// Flag if the default feature should be built for the root package
     pub no_default_features: bool,
     /// Root package to build (if None it's the current one)
-    pub spec: Option<&'a str>,
+    pub spec: Vec<&'a str>,
```

Comment:
This may be much easier to take if it's a `&'a [String]`, that way all the commands above can avoid casting and collection (e.g. they're just `&options.flag_package`